src/wp-includes/option.php

```diff
@@ -277,6 +277,19 @@ function form_option( $option ) {
 function wp_load_alloptions( $force_cache = false ) {
 	global $wpdb;
 
+	/**
+	 * This filter allows the wp load all options function to be shortcut.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param null|array $pre if an array is returned then the wp_load_alloptions will be shortcuted and the array returned.
+	 * @param bool       $force_cache Whether to force an update of the local cache from the persistent cache. Default false.
+	 */
+	$pre = apply_filters( 'pre_wp_load_alloptions', null, $force_cache );
+	if ( null !== $pre ) {
+		return $pre;
+	}
```

Comment:
I think we can use a more descriptive name for this variable. 
```suggestion
	$alloptions = apply_filters( 'pre_wp_load_alloptions', null, $force_cache );
	if ( null !== $alloptions ) {
		return $alloptions ;
	}
```